src/main/java/org/javarosa/xform/parse/XFormParser.java

```diff
@@ -182,6 +183,9 @@ public class XFormParser implements IXFormParserFunctions {
     private final List<FormDefProcessor> formDefProcessors = new ArrayList<>();
     private final List<ModelAttributeProcessor> modelAttributeProcessors = new ArrayList<>();
     private final List<QuestionProcessor> questionProcessors = new ArrayList<>();
+    private final List<XPathProcessor> xpathProcessors = new ArrayList<>();
+
+    public static final List<XPathProcessor> tempXPathProcessors = new ArrayList<>();
```

Comment:
Took me a bit to understand that this is just so the processors can be accessed statically even though they need to be set on an instance to use the existing `addProcessor`.